game/src/main/resources/assets/sol_client/lang/vi_VN.lang

```diff
@@ -0,0 +1,439 @@
+
+sol_client.mod.freelook.name=Freelook
+sol_client.mod.freelook.description=Tự do nhìn xung quanh.
+
+sol_client.mod.freelook.key=Freelook
+sol_client.mod.freelook.option.key=Phím tắt
+
+sol_client.mod.freelook.option.perspective=Hướng nhìn
+sol_client.mod.freelook.option.invertPitch=Đảo ngược hướng dọc của chuột
+sol_client.mod.freelook.option.invertYaw=Đảo ngược hướng ngang của chuột
+
+sol_client.mod.taplook.name=Taplook
+sol_client.mod.taplook.description=Chuyển đổi góc nhìn nhanh bằng phím tắt.
+
+sol_client.mod.taplook.key=Taplook
+sol_client.mod.taplook.option.key=Phím tắt
+sol_client.mod.taplook.option.perspective=Góc nhìn khi ấn
+
+sol_client.mod.toggle_sprint.name=Toggle Sprint
+sol_client.mod.toggle_sprint.description=Khoá phím chạy.
+
+sol_client.mod.toggle_sprint.default_width=113
+
+sol_client.mod.toggle_sprint.option.hud=HUD
+
+sol_client.mod.toggle_sprint.held=Đang chạy (Giữ)
+sol_client.mod.toggle_sprint.toggled=Đang chạy (Khoá phím)
+
+sol_client.mod.zoom.name=Zoom
+sol_client.mod.zoom.description=Phóng to màn hình
+
+sol_client.mod.zoom.key=Phóng to
+sol_client.mod.zoom.option.key=Phím tắt
+sol_client.mod.zoom.zoom_out=Thu nhỏ
+sol_client.mod.zoom.option.zoomOutKey=Phím tắt thu nhỏ
+sol_client.mod.zoom.zoom_in=Phóng to
+sol_client.mod.zoom.option.zoomInKey=Phím tắt phóng to
+sol_client.mod.zoom.option.cinematic=Camera kịch tính
+sol_client.mod.zoom.option.reduceSensitivity=Giảm tốc độ chuột
+sol_client.mod.zoom.option.scrolling=Thay đổi độ lớn bằng cuộn chuột
+sol_client.mod.zoom.option.smooth=Hiệu ứng thu phóng
+sol_client.mod.zoom.option.factor=Độ lớn thu phóng
+
+sol_client.mod.replay.name=Replay
+sol_client.mod.replay.description=Ghi lại diễn biến trò chơi và xem lại khi bạn muốn.
+
+sol_client.mod.replay.option.enableNotifications=Thông báo hoạt động
+sol_client.mod.replay.option.recordSingleplayer=Ghi hình chơi đơn
+sol_client.mod.replay.option.recordServer=Ghi hình chơi nhiều người
+sol_client.mod.replay.option.recordingIndicator=Hiển thị Chỉ báo Ghi hình
+sol_client.mod.replay.option.recordingIndicatorColour=Màu sắc Chỉ báo
+sol_client.mod.replay.option.recordingIndicatorScale=Kích thước Chỉ báo
+sol_client.mod.replay.option.recordingIndicatorTextColour=Màu sắc chữ
+sol_client.mod.replay.option.recordingIndicatorTextShadow=Bóng đổ chữ
+sol_client.mod.replay.option.automaticRecording=Tự động bắt đầu ghi hình
+sol_client.mod.replay.option.renameDialog=Cửa sổ đổi tên bản ghi khi hoàn tất ghi hình
+sol_client.mod.replay.option.showChat=Hiển thị Chat
+
+sol_client.mod.replay.option.camera=Camera
+sol_client.mod.replay.option.camera.classic=Classic
+sol_client.mod.replay.option.camera.vanilla_ish=Vanilla-ish
+
+sol_client.mod.replay.option.showPathPreview=Show Path Preview
+
+sol_client.mod.replay.option.defaultInterpolator=Default Interpolator
+sol_client.mod.replay.interpolator.catmull=Catmull-Rom Spline
+sol_client.mod.replay.interpolator.cubic=Cubic Spline
+sol_client.mod.replay.interpolator.linear=Linear
```

Comment:
Not translated.

Reply:
ReplayMod has a lot of technical words that have no translation in English, so I decided to let them all be untranslated instead of a screen half English and half Vietnamese.

Reply:
I'd prefer for as much as possible to be translated.